KarhooUISDK/CommonUI/LoyaltyInformation/LoyaltyInformationView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  LoyaltyInformationView.swift
+//  KarhooUISDK
+//
+//  Created by Aleksander Wedrychowski on 23/11/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+/// View presenting user-friendly information how many poins shall be added/removed to/from Loyalty.
+struct KarhooLoyaltyInformationView: View {
+
+    private enum Constants {
+        static let iconSide: CGFloat = 16
+    }
+
+    // MARK: - Properties
+
+    let viewModel: ViewModel
```

Comment:
Why it is not `KarhooLoyaltyInformationViewModel` Don't you afraid it will be mass with initialisation?